scst-store/deployment-details.hbs.md

```diff
@@ -15,7 +15,7 @@ The installation creates the following in your Kubernetes cluster:
     - `metadata-store-app`
     - `metadata-store-db`
     - `amr-cloudevent-handler`. If AMR is deployed, see [Deploying AMR](#amr).
-    - `artifact-metadata-repository-app`. If AMR is deployed, see [Deploying AMR](#amr).
+    - `amr-graphql-app`. If AMR is deployed, see [Deploying AMR](#amr).
```

Comment:
Isn't AMR deployed by default now? Shouldn't this note go away? I believe it now makes references to value fields that no longer exist. Or is this meant to be handled in another later story?

Reply:
Technically that should be handled in a separate story. I'll coordinate with Tony to make sure the docs reflect that.

Reply:
A new story has been created to address this issue.